### PR TITLE
"Event" instead of "New event" to fit new design (since Nextcloud 27)

### PR DESCRIPTION
**Pull request by Jerome-Herbinet**
## Currently in French (longer words than in English) in Nextcloud 27 RC1 :

Note : It may not affect negatively previous major versions (just a shorter wording)

## After, in 3 languages : 
2023-05-24_11-55-FR
2023-05-24_11-54-DE
2023-05-24_11-55-EN

@nextcloud/designers

### CODE REVIEW

File: src/components/AppNavigation/AppNavigationHeader/AppNavigationHeaderNewEvent.vue [hunk 1]
```diff
@@ -29,7 +29,7 @@
 		<template #icon>
 			<Plus :size="20" />
 		</template>
-		{{ $t('calendar', 'New event') }}
+		{{ $t('calendar', 'Event') }}
```
Comment (nickvergessen) [inline src/components/AppNavigation/AppNavigationHeader/AppNavigationHeaderNewEvent.vue:32]: The button should have a title or aria-label with a more descriptive text than "Event"

Comment (Jerome-Herbinet) [reply src/components/AppNavigation/AppNavigationHeader/AppNavigationHeaderNewEvent.vue:34]: Understood @nickvergessen. 
Please check my additional commit. 
It's the first time I managed that. 
Is it correctly done and enough ?

File: src/components/AppNavigation/AppNavigationHeader/AppNavigationHeaderNewEvent.vue [hunk 2]
```diff
@@ -22,14 +22,16 @@
 
 <template>
 	<NcButton v-shortkey="['c']"
+		:aria-label="newEvent"
 		class="button new-event"
+		:title="newEvent"
```
Comment (tcitworld) [inline src/components/AppNavigation/AppNavigationHeader/AppNavigationHeaderNewEvent.vue:27]: **issue:** Some screen-readers read **both** `aria-label` and `title`, so you don't want to put both. I think having only `aria-label` is more appropriate.

Comment (Jerome-Herbinet) [reply src/components/AppNavigation/AppNavigationHeader/AppNavigationHeaderNewEvent.vue]: @tcitworld check my 3rd commit :-)

File: src/components/AppNavigation/AppNavigationHeader/AppNavigationHeaderNewEvent.vue [hunk 3]
```diff
@@ -51,6 +52,11 @@ export default {
 			this.$router.push(`/new/${this.$route.params.view}`)
 		},
 	},
+	computed: {
+		newEvent() {
```
Comment (nickvergessen) [inline src/components/AppNavigation/AppNavigationHeader/AppNavigationHeaderNewEvent.vue:56]: Duplicated key 'newEvent'

Comment (Jerome-Herbinet) [reply src/components/AppNavigation/AppNavigationHeader/AppNavigationHeaderNewEvent.vue]: OK ; Check 4th commit @nickvergessen

**Review by tcitworld — APPROVED** [review]

File: src/components/AppNavigation/AppNavigationHeader/AppNavigationHeaderNewEvent.vue [hunk 4]
```diff
@@ -51,6 +52,11 @@ export default {
 			this.$router.push(`/new/${this.$route.params.view}`)
 		},
 	},
+	computed: {
+		newEventButton() {
```
Comment (nickvergessen) [inline src/components/AppNavigation/AppNavigationHeader/AppNavigationHeaderNewEvent.vue:56]: **Optional:**
```suggestion
		newEventButtonAriaLabel() {
```

And afterwards squash all :P

File: src/components/AppNavigation/AppNavigationHeader/AppNavigationHeaderNewEvent.vue [hunk 5]
```diff
@@ -51,6 +52,11 @@ export default {
 			this.$router.push(`/new/${this.$route.params.view}`)
 		},
 	},
+	computed: {
+		newEventButtonAriaLabel() {
```
[the remark this answers is not in the document]
Comment (Jerome-Herbinet) [reply src/components/AppNavigation/AppNavigationHeader/AppNavigationHeaderNewEvent.vue]: Can you check my 2 last commits before I squash ?

[the remark this answers is not in the document]
Comment (nickvergessen) [reply src/components/AppNavigation/AppNavigationHeader/AppNavigationHeaderNewEvent.vue]: Looks good I guess.

**Comment by Jerome-Herbinet** [comment]
6 commits squashed in one. 
Please have a last look to establish if everything is OK for you.

**Review by st3iny — APPROVED** [review]
Tested and works. Can't be backported due to translations changes.
